src/Stratis.Bitcoin.Features.MemoryPool/MempoolValidator.cs

```diff
@@ -151,32 +154,32 @@ public class MempoolValidator : IMempoolValidator
 
         private Network network;
 
+
```

Comment:
Empty line?

Reply:
I'll remove, but worth noting there is commented code just above this is not good either and needs promoting to an issue/task.

However this isn't relevant to your comment

```
        // TODO: Implement Later with CheckRateLimit()
        //private readonly FreeLimiterSection freeLimiter;

        //private class FreeLimiterSection
        //{
        //  public double FreeCount;
        //  public long LastTime;
        //}
```

Reply:
Agree